### PR TITLE
fix: add dhrumishah

### LINKED ISSUE #1203: Dhrumi Shah
Screenshot of my profile

image

### PR DESCRIPTION
**Pull request by dhrumishah**
resolves #1203

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey! please add a line blank line below!!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by dhrumishah** [comment]
Is it okay now? @Pradumnasaraf

File: profiles/dhrumishah.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/84569241/175770956-55116be8-46cb-470c-99b2-1b597158e1c6.png)",
+    "issueId": 1203,
+    "name": "Dhrumi Shah",
+    "username": "dhrumishah"
+}
```
Comment (Pradumnasaraf) [inline profiles/dhrumishah.json:6]:
```suggestion
{
  "image": "https://user-images.githubusercontent.com/84569241/175770956-55116be8-46cb-470c-99b2-1b597158e1c6.png)",
  "issueId": 1203,
  "name": "Dhrumi Shah",
  "username": "dhrumishah"
}
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thank you for the previous changes @dhrumishah, there is still a small indentation issue that the linter caught and an extra parenthesis. Please add those changes, or you also simply do `commit suggestions`

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

**Comment by dhrumishah** [comment]
> Thank you for the previous changes @dhrumishah, there is still a small indentation issue that the linter caught and an extra parenthesis at the end `https:......png)`. Please add those changes, or you also simply do `commit suggestions`
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix  |  **Tags:** Not reusable

I have resolved the above changes. please do let me know if it has any error.

**Review by Pradumnasaraf — APPROVED** [review]
Thank you for the changes. All checks are now passed.
Looks good to merge :+1: 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

**Pull request merged**